index.html

```diff
@@ -453,52 +444,60 @@ <h3>
           </li>
         </ul>
         <p>
-          The <dfn>text-direction values</dfn> are the following, implying that
-          the value of the <a>localizable members</a> is by default:
+          The <dfn>text-directions</dfn> are the following, implying that the
+          value of the <a>localizable members</a> is by default:
         </p>
         <dl>
           <dt>
-            "<dfn>ltr</dfn>"
+            "<dfn data-dfn-for="text-direction">ltr</dfn>"
           </dt>
           <dd>
             Left-to-right text.
           </dd>
           <dt>
-            "<dfn>rtl</dfn>"
+            "<dfn data-dfn-for="text-direction">rtl</dfn>"
           </dt>
           <dd>
             Right-to-left text.
           </dd>
           <dt>
-            "<dfn>auto</dfn>" (default)
+            "<dfn data-dfn-for="text-direction">auto</dfn>" (default)
           </dt>
           <dd>
             No explicit directionality.
           </dd>
         </dl>
+        <p>
+          The <dfn>text-direction list</dfn> is the [=list=] «
+          "[=text-direction/ltr=]", "[=text-direction/rtl=]",
+          "[=text-direction/auto=]" ».
+        </p>
         <p>
           When displaying the <a>localizable members</a> to an end-user, the
           use agent SHOULD:
         </p>
         <ol class="algorithm">
-          <li>If the <a>base direction</a> is <a>ltr</a> or <a>rtl</a>,
-          override <a data-cite="bidi#P3">Rule P3</a> of [[BIDI]], setting the
-          paragraph embedding level to 0 if the <a>base direction</a> is
-          <a>ltr</a>, or 1 if the <a>base direction</a> is <a>rtl</a>.
+          <li>If the <a>base direction</a> is [=text-direction/ltr=] or
+          [=text-direction/rtl=], override <a data-cite="bidi#P3">Rule P3</a>
+          of [[BIDI]], setting the paragraph embedding level to 0 if the
+          <a>base direction</a> is [=text-direction/ltr=], or 1 if the <a>base
+          direction</a> is [=text-direction/rtl=].
           </li>
-          <li>Otherwise the <a>base direction</a> is "[=auto=]", in which case
-          determine the text's direction by applying <a data-cite=
-          "bidi#P1">Rule P1</a> of [[BIDI]].
+          <li>Otherwise the <a>base direction</a> is "[=text-direction/auto=]",
+          in which case determine the text's direction by applying
+            <a data-cite="bidi#P1">Rule P1</a> of [[BIDI]].
           </li>
         </ol>
         <p>
-          To <dfn>process the `dir` member</dfn>, given [=object=] |json:JSON|
-          and [=ordered map=] |manifest:ordered map|:
+          To <dfn>process the `dir` member</dfn>, given [=ordered map=]
```

Comment:
*the* ordered map*s* ?

Reply:
I think this one is ok. It gets rendered as :

"To process the `dir` member, given ordered map _json_ and ordered map _manfiest_:"

Reply:
hopefully not *man fiest* :-D